src/main/java/Duke.java

```diff
@@ -29,9 +29,32 @@ public static void main(String[] args) {
                     todo.get(Integer.parseInt(num)-1).markAsDone();
                     System.out.println(num + ". " + "[" + todo.get(Integer.parseInt(num)-1).getStatusIcon() +"] " + todo.get(Integer.parseInt(num)-1).getDescription());
                 } else {
-                    Task t = new Task(readtext);
-                    todo.add(t);
-                    System.out.println("added: "  + readtext);
+                    if (readtext.contains("todo")) {
+                        String spli = readtext.split("todo")[1];
+                        Task t = new Todo(spli);
+                        todo.add(t);
+                        System.out.println("Got it. I've added this task");
+                        System.out.println(t.toString());
+                        System.out.println("Now you have " + todo.size() + " tasks in the list");
+                    } else if (readtext.contains("deadline")) {
+                        String spli = readtext.split("deadline")[1];
+                        String des = spli.split("/by")[0];
```

Comment:
This is minor but maybe it’ll be better with a clear naming of variables, instead of short form des and fin.